libs/service-portal/documents/codegen.yml

```diff
@@ -0,0 +1,18 @@
+schema:
+  - apps/api/src/api.graphql
+documents:
+  - libs/service-portal/documents/src/**/**/*.graphql
+generates:
+  libs/service-portal/documents/src/:
+    preset: 'near-operation-file'
+    presetConfig:
+      baseTypesPath: '~@island.is/api/schema'
+    plugins:
+      - typescript-operations
+      - typescript-react-apollo
+    config:
+      scalars:
+        DateTime: string
```

Comment:
I know we repeatedly convert the graphql datetimes to string in the config, but why?

Reply:
I'm going to remove this.